(social-connections/facebook) update guide (#1860)
clerk/clerk-docs: docs/authentication/social-connections/facebook.mdx

1. In the Clerk Dashboard, navigate to the [**SSO connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
1. Select **Add connection** and select **For all users**.
1. In the **Choose provider** dropdown, select **Facebook**.
1. Select **Add connection**.

## Configure for your production instance

For _production instances_, you must provide custom credentials.

To make the setup process easier, it's recommended to keep two browser tabs open: one for the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) and one for your [Facebook Developer Console](https://developers.facebook.com).

<Steps>
### Enable Facebook as a social connection

1. In the Clerk Dashboard, navigate to the [**SSO connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
1. Select **Add connection** and select **For all users**.
1. In the **Choose provider** dropdown, select **Facebook**.
1. Ensure that both **Enable for sign-up and sign-in** and **Use custom credentials** are toggled on.

> 1. Save the **Redirect URI** somewhere secure. Keep this modal and page open.

Reviewer: ```suggestion
1. Save the **Valid OAuth Redirect URI** somewhere secure. Keep this modal and page open.
```

### Create a Facebook app

1. In the top-right of the Facebook Developer page, select [**My Apps**](https://developers.facebook.com/apps).
1. In the top-right, select **Create App**. You'll be redirected to the **Create an app** process.
    1. In the **App details** step, fill out the necessary information and select **Next**.
    1. In the **Use Cases** step, select **Authenticate and request data from users with Facebook Login** and then select **Next**.
    1. In the **Business** step, select the business portfolio to connect to your app and then select **Next**.
    1. In the **Finalize** step, select **Go to dashboard**. You'll be redirected to the app's **Dashboard** page.
1. In the left sidebar, select **Use cases**.
1. Next to **Authenticate and request data from users with Facebook Login**, select **Customize**. You'll be redirected to the **Permissions** tab of the **Customize use case** page.
1. Next to **email**, select **Add**. This permission allows Clerk to read your user's primary email address.
1. In the left sidebar, under **Facebook Login**, select **Settings**.

> 1. In the **Client OAuth settings** section, in the **Valid OAuth Redirect URIs** field, paste the **Redirect URI** value you saved from the Clerk Dashboard.

Reviewer: ```suggestion
1. In the **Client OAuth settings** section, in the **Valid OAuth Redirect URIs** field, paste the **Valid OAuth Redirect URI** value you saved from the Clerk Dashboard.
```

1. Select **Save changes**.

> 1. In the left sidebar, select **App settings** (or the settings icon) and thenselect **Basic**.

Reviewer: ```suggestion
1. In the left sidebar, select **App settings** (hover over the icon to view the title or expand the menu), and then select **Basic**.
```
